src/common/snippets/include/snippets/lowered/linear_ir.hpp

```diff
@@ -21,6 +21,14 @@ class Config {
     // True if we should check runtime info for nodes to call specific needed transformations
     bool m_need_fill_tail_register = false;
     size_t m_loop_depth = 1;
+    // Some Subgraphs doesn't support domain optimization due to operations' semantics
+    bool m_enable_domain_optimization = false;
+    // Minimal advised work amount for parallel execution.
+    // Set by a backend, typically equals to the number of threads available on the machine.
+    size_t m_min_parallel_work_amount = 8;
+    // Minimal advised work amount every JIT kernel should process during one execution call
+    // Set by a backend, should be large enough to compensate for the kernel call overheads
+    size_t m_min_jit_work_amount = 256;
```

Comment:
"jit" wording seems strange here. It is kind of internal code generation method. 
More accurate explanation is that this param adjusts minimal work_amount for executable returned by Subgraph::generate

Reply:
Fair enough, corrected.